tortoisebeacon/tortoisebeacon.go

```diff
@@ -182,94 +182,34 @@ func (tb *TortoiseBeacon) Close() error {
 	return nil
 }
 
-// Get returns a Tortoise Beacon value as types.Hash32 for a certain epoch or an error if it doesn't exist.
-// TODO(nkryuchkov): Consider making unexported.
-func (tb *TortoiseBeacon) Get(epochID types.EpochID) (types.Hash32, error) {
+// Get returns a Tortoise Beacon value as []byte for a certain epoch or an error if it doesn't exist.
+func (tb *TortoiseBeacon) Get(layerID types.LayerID) ([]byte, error) {
```

Comment:
why `[]byte` ?

Reply:
It's easier to mock. However, I'm still in doubt whether it should be `types.Hash32` or `[]byte`